profiles/sink/Sink_4K_Display.yaml

```diff
@@ -3,6 +3,8 @@ dsDisplay:
   Name: TV
   features:
     extendedEnumsSupported: false
+  Number_of_ports: 1
+  Video_Ports: 0x08 # dsVIDEOPORT_TYPE_INTERNAL
```

Comment:
use array([0x08]) for Video_Ports and read as array in the code.

Reply:
done